docs/reference/3-admin-api/installation/admin-api-1x-for-odsapi-5x-6x/admin-api-1x-iis-installation-powershell.md

```diff
@@ -0,0 +1,417 @@
+
+There will be two folders. AdminApi folder will have binaries. Installer folder
+contains PowerShell scripts required for installation.
+
+### Step 2. Configure Installation
+
+Open the "install.ps1" file in a text editor. You will need to edit this file
+with your configuration details. If a value is not present for any of the
+parameters, it will use its default value.
+
+**Note: Editing Items 3(a, b) and 4b below are mandatory for installation to
+complete.**
+
+1. Configure `$dbConnectionInfo`. These values are used to construct the
+   connection strings.
+    1. `Server`. The name of the database server. For a local server, we can use
+       "(local)" for SQL and "localhost" for PostgreSQL.
+
+    2. `Engine.` Admin App supports SQL and PostgreSQL database engines. So
+       setting up the `Engine` will decide which database engine to be used.
+       Valid values are "SQLServer" and "PostgreSQL".
+    3. `UseIntegratedSecurity.` Will either be "$true" or "$false".
+        1. If you plan to use Windows authentication, this value will be "$true"
+        2. If you plan to use SQL Server/ PostgreSQL server authentication, this
+           value will be "$false" and the Username and `Password` must be
+           provided.
+    4. `Username`. Optional. The username to connect to the database.
+       If `UseIntegratedSecurity` is set to $true, this entry is not needed
+    5. `Password`. Optional. The password to connect to the
+       database. If `UseIntegratedSecurity` is set to $true, this entry is not
+       needed
+    6. `Port.` Optional. Used to specify the database server port, presuming the
+       server is configured to use the specific port.
+2. Configure `$adminAppFeatures`. These values are used to set Optional
+   overrides for features and settings in the appsetting.json.
+    1. `ApiMode.` Possible
+       values: `sharedinstance`, `districtspecific` and `yearspecific`. Defaults
+       to `sharedinstance`
+3. Configure `$authenticationSettings`. These values are mandatory for
+   authentication process.
+
+               a. `SigningKey:` must be a Base64-encoded string
+               b. `Authority and IssuerUrl:` should be the same URL as your application
+               c. `AllowRegistration:` to true allows unrestricted registration of new Admin API clients. This is not recommended for production.
+
+     4. Configure `$p`. This is the variable used to send all the information to
+     the installation process.
+
+1. 1. `ToolsPath`. Path for storing installation tools, e.g., nuget.exe.
+      Defaults to "C:/temp/tools"
+   2. **`OdsApiUrl`. Base URL for the ODS / API. Mandatory.**
+   3. `PackageVersion`. Optional. If not set, will retrieve the latest full
+      release package.
+
+> [!NOTE] ![Screenshot](https://edfi.atlassian.net/wiki/images/icons/grey_arrow_down.png)
+>
+> SQL Server
```

Comment:
If you rename this file as `.mdx` instead of `.md`, you can implement [tabs](https://docusaurus.io/docs/3.5.2/markdown-features/tabs) for SQL Server and PostgreSQL. [Example](https://docs.ed-fi.org/reference/ods-api/getting-started/binary-installation/sandbox-installation-steps) ([source code](https://github.com/Ed-Fi-Alliance-OSS/ed-fi-alliance-oss.github.io/blob/main/odsApi_versioned_docs/version-7.2/getting-started/binary-installation/sandbox-installation-steps.mdx))